packages/discord.js/src/util/Util.js

```diff
@@ -315,8 +315,8 @@ function basename(path, ext) {
  * @returns {string}
  */
 function cleanContent(str, channel) {
-  return str.replaceAll(/<(@[!&]?|#)(\d{17,19})>/g, (match, type, id) => {
-    switch (type) {
+  return str.replaceAll(/<(?:(@[!&]?|#)|([/:])([\w ]+):)(\d{17,19})>/g, (match, type, prefix, name, id) => {
```

Comment:
Your regexp does not match animated emojis! it's also became much more complex to handle.

![Google Chrome Dev - 2023-03-29 at 17 03 58@2x](https://user-images.githubusercontent.com/17960496/228563557-12db4694-f89b-4e11-b06e-1ac1be1700fc.png)

You could use `<(?:(?<type>@[!&]?|#)|(?:(?<command>\/[\w ]+)|(?<animatedEmoji>a)?:(?<emojiName>[\w]+)):)(?<id>\d{17,19})>` for a regex, and then use the groups for the actual replacing (if you prefer that), or.. honestly 🤷 whatever floats your boat

![Google Chrome Dev - 2023-03-29 at 17 14 12@2x](https://user-images.githubusercontent.com/17960496/228566691-a8c7e2f2-bbf9-48bb-9524-d79f18583992.png)